content/200-orm/100-prisma-schema/20-data-model/90-multiple-files.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Multi-file Prisma schema
+metaTitle: Prisma schema with multiple files
+metaDescription: Learn how to split your Prisma schema into multiple files.
+---
+
+:::tip
+
+Multi-file Prisma schema is available via the `prismaSchemaFolder` preview feature in Prisma versions 5.15.0 and later.
+
+:::
+
+## How to enable multi-file Prisma schema support
+
+Multiple Prisma schema file support is currently in preview. To enable the feature, add the `prismaSchemaFolder` feature flag to the `previewFeatures` field of the `generator` block in your Prisma schema: 
+
+```prisma file=schema.prisma showLineNumbers
+generator client {
+  provider        = "prisma-client-js"
+  //add-next-line
+  previewFeatures = ["prismaSchemaFolder"]
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+## How to include multiple Prisma schema files in your project
+
+To use multiple Prisma schema files, add a `schema` inside of your current `prisma` directory. With the `prismaSchemaFolder` Preview feature enabled, you can add as many files as you want to the `prisma/schema` directory.
+
+## How to use existing Prisma CLI commands with multiple Prisma schema files
+
+Existing Prisma CLI commands have been updated to support multiple Prisma files. Simply replace references to a file with a directory. As an example, the following `prisma migrate diff` command:
+
+```terminal
+npx prisma migrate diff \
+--from-empty \
+--to-schema-datamodel prisma/schema.prisma \
+--script > prisma/migrations/0_init/migration.sql
+```
+
+becomes the following:
+
+```terminal
+npx prisma migrate diff \
+--from-empty \
+//highlight-next-line
+--to-schema-datamodel prisma/schema \
+--script > prisma/migrations/0_init/migration.sql
+```
+
+## Best practices
```

Comment:
Nit: Please don't call them "Best Practices." That would be too close to "conventions," and then people would say that this is our recommendation. These are really just tips on how to make the most of the feature, right?